rfcs/0050-roadmaps.md

```diff
@@ -0,0 +1,320 @@
+  a `tvm:roadmap` tag so that they can be properly triaged and filtered.
+
+## Rationale & Alternatives
+
+There are several alternatives to this roadmap proposal; however, each of the alternatives has
+drawbacks of its own.
+
+- **Do nothing:** Don’t create a publicly viewable roadmap. However, the community would miss out on
+  the numerous benefits of having a roadmap.
+- **Create a separate repository for roadmap:** Having a separate roadmap repository
+  (e.g. `apache/tvm-roadmap`) might reduce the number of roadmap issues filed directly to
+  `apache/tvm`, but it would increase the complexity and fragmentation of the roadmap significantly,
+  since this roadmap repository would have to link to RFC tracking issues in `apache/tvm` anyways.
+- **Use a separate tool for the roadmap:** Using a separate tool for the roadmap may provide more
+  functionality than Github projects. However, this would require much more effort to create and
+  maintain.
+
+## Prior Art
+
+This RFC is inspired by the previous work done to improve the tracking mechanisms of TVM. This
+includes but is not limited to the existing RFC process, release process, and technical visions of
+TVM.
+
+![image|690x351](./assets/0050/prior-art-1.jpeg)
+
+It is also inspired by the [public Github roadmap](https://github.com/github/roadmap), which is an
+excellent example of usage for Github Projects.
+
+![image|690x426](./assets/0050/prior-art-public-gh-roadmap.png)
+
+## Upstreaming Timeline
+
+This is a prospective upstreaming timeline for the TVM roadmap.
+
+|Milestone Name|Deliverables|Schedule|
+| --- | --- | --- |
+|TVM Roadmap M1A: Taking the Plunge|<li>All roadmap projects defined<br><li>Initial prototype roadmap (TVM CI & Testing) upstreamed <br> <li>Skeleton roadmap documentation upstreamed <br> <li>Scope of M1B and M1C clearly defined|Late September|
+|TVM Roadmap M1B: Building Momentum|<li> At least 3 additional roadmaps upstreamed <br> <li> Additional roadmap process documentation upstreamed|Mid October|
+|TVM Roadmap M1C: Ready for Prime Time|<li> All initial roadmaps (described in M1A RFC) upstreamed <br> <li> All RFC opens from M1A and M1B addressed <br><li> Publicize roadmap in documentation and TVM forums|Early November|
+
+## Unresolved Questions
+
+Questions about the design and contents of the TVM Roadmap will be progressively resolved through
+the upstreaming timeline shown above. Any unresolved questions at the end of TVM Roadmap Milestone 1
+will be created as roadmap items in the TVM Community Roadmap 🙂
+
+## Future Possibilities
+
+- Create [Github issue templates](https://github.com/apache/tvm/tree/main/.github/ISSUE_TEMPLATE)
+  for roadmap items (similar to the existing Github issue templates for docs, CI, etc).
+- Use [the new Github issues/projects (closed beta)](https://github.com/features/issues) to improve
+  roadmap functionality.
+
+## Special Thanks
+Thanks so much for copyediting, @areusch and @electriclilies! :hugs:
```

Comment:
Should this be @denise-k and @electriclilies given @areusch is the author here? :smile_cat:

Reply:
thanks for bringing this up - I authored this initially (which is why I didn't refer to myself in third person), and @areusch ported it over to this RFC + made some additions. @areusch and I are already both listed as coauthors, so really it should be just @electriclilies listed as a copyeditor.